### PR TITLE
JK PS3

### PR DESCRIPTION
**Pull request by johndkim89**
[John_Kim_Problem3.pdf](https://github.com/macss-ml20/problem-set-3/files/4216524/John_Kim_Problem3.pdf)

### CODE REVIEW

**Comment by johndkim89** [comment]
[John_Kim_PS3.pdf](https://github.com/macss-ml20/problem-set-3/files/4216359/John_Kim_PS3.pdf)

**Comment by johndkim89** [comment]
[John_Kim_Problem3.pdf](https://github.com/macss-ml20/problem-set-3/files/4216521/John_Kim_Problem3.pdf)

**Comment by rrodriguezbarron** [comment]
Evaluation
----------------------------------------------------------------------------
| Topic					 	|Total	| Score | Comments |
|-----------------------------------------------|-------|-------|----------|
| Decision Trees				|	|	||
| **Create Training Set**			|10	|10	|`df` is not an appropriate name for a named item. Consider a more descriptive one.|
| **Plot training and test set** 		|15	|8	|Only one of the sets was plotted. *No points were deduced for this.*|
| **Update lambda, retest and discuss** 	|10	|10	||
| **Apply bagging to the training set** 	|10	|10	||
| **Apply random forest to training set**	|10	|10	||
| **Apply linear regression to training set** 	| 5	| 5	||
| **Compare test errors across all fits**	| 5	| 0	|Wrong conclusions. |
| Support Vector Machines			|	|	||
| **Fit a support vector classifier**		|10	|5	|X should be scaled.|
| **Display the confusion matrix**		| 5	| 5	||
| **Find an optimal cost**			|10	|10	||
| **Compute and discuss differences**		|10	|10	||
| **Penalty for late submission**            		|-10	|-10	|P.Waggoner 2020-02-28|
| **TOTAL**            				|90	|73	||

Remarks:
* 10 point penalty
* MSE Calculation and Method Evaluation: Numbers in items were not correctly assessed or maybe wrong comparison. Either way, my suggestion is to review the concepts seen in class for comparing MSEs.
